tests/TypeApiTests.hs

```diff
@@ -43,26 +43,26 @@ getQuery query =
         parseOnly (document <* endOfInput) query
   in selectionSet
 
-runQuery :: AST.SelectionSet -> IO (Either Text Value)
+runQuery :: AST.SelectionSet -> IO (Either Text Result)
```

Comment:
Sorry, I don't quite understand. I get that we're running over a user-level `ExceptT` in this test, but maybe we should not do that any more, since nothing in the code makes use of it?